src/hooker/hooker.cpp

```diff
@@ -37,3 +39,34 @@ void StopHooking()
     DWORD OldProtect2;
     VirtualProtectEx(GetCurrentProcess(), (LPVOID)0x00401000, nBinarySize, OldProtect, &OldProtect2);
 }
+
+bool Check_Pointer(uintptr_t address, char *data)
+{
+    static const char *patterns[] = {
+        "\x55\x89\xE5", //watcall and stdcall
+        "\x53\x83", //cdecl
+        "\x53\x51", //cdecl
+        "\x8B\x44", //cdecl
+        nullptr
+    };
+
+    bool found = false;
+
+    int i = 0;
```

Comment:
`i` isn't used outside the scope so it should be `for (int i = 0; i < ARRAY_SIZE(patterns); ++i)`